### ISSUE TITLE
Need For Speed: ProStreet BLUS30066 fatally crashes on track load

### ISSUE THREAD

**Issue by superjeepboy** [RPCS3/rpcs3#9885]
## Quick summary
Please briefly describe what is not working correctly.

Game attempts to load into a world, but promptly terminates.

## Details

Please describe the problem as accurately as possible.

Starting a career file or even loading into a map immediately crashes the game.
[RPCS3.log](https://github.com/RPCS3/rpcs3/files/6074470/RPCS3.log)

#### 1. Please attach RPCS3's log.
* Run the game until you find the issue.
* Completely close RPCS3, or move to the next step in case it has crashed.
* Locate RPCS3's log file:
	+ ```RPCS3.log``` (It can show up as just RPCS3 and have a notepad icon)
	or
	+ ```RPCS3.log.gz``` (It can show up as RPCS3.log and have a zip or rar icon)
![image](https://user-images.githubusercontent.com/44116740/84433247-aa15fa80-ac36-11ea-913e-6fe25a1d040e.png)
	On Linux it will be in ```~/.cache/rpcs3/```.
	On Windows it will be near the executable.
* Attach the log:
	+ Drag and drop your log file into the issue.
	+ Or upload it to the cloud, such as Dropbox, Mega etc. 

#### 2. If you describe graphical issue, please provide an RSX capture and a RenderDoc capture that demonstrate it.
* To create an RSX capture, use _Create_ _RSX_ _Capture_ under _Utilities_.
Captures will be stored in RPCS3 folder → captures.
	+ Compress your capture with 7z, Rar etc. 
	And	drag and drop it into the issue.
	+ Or upload it to the cloud, such as Dropbox, Mega etc.
* To create a RenderDoc capture, please refer to [RenderDoc's documentation](https://renderdoc.org/docs/how/how_capture_frame.html).

#### 3. Please attach screenshots of your problem.
* Enable performance overlay with at least medium level of detail.
You can find it in _Emulator_ tab in _Settings_.
![rpcs3_JmPwbQQhd4](https://user-images.githubusercontent.com/5252182/109775118-f4b91780-7bb5-11eb-845b-e43f887b7207.png)

#### 5. Please provide your system configuration:
* OS: Windows 10
* CPU: AMD Ryzen 5 3600X
* GPU: MSI Radeon 5700XT MECH OC
* Driver version: 21.2.1
* etc.

#### Please include.
* Anything else you deem to be important

**Comment by Asinin3** [comment]
I can get ingame without issue on latest, closing.
![rpcs3_Vho6bEwy7z](https://user-images.githubusercontent.com/15137035/164571302-c89df8e7-358a-4d74-b595-2dfa3f06846c.jpg)
Let us know if its still an issue

**Comment by Satan86** [comment]
I don't think this issue should be closed, just renamed instead, because while it can go past the loading, it still crashes with the same error later on in gameplay
![image](https://user-images.githubusercontent.com/61447828/164674678-d42667e9-bbc5-4513-8e22-8d34c6ed943b.png)

**Comment by YuriNator557** [comment]
Has SPU ASMJIT been tried?

**Comment by hoholee12** [comment]
ASMJIT does nothing to fix the error.

**Comment by cas-1092** [comment]
Setting "PPU Thread Count" to 1 in debug tab fixes the random crashing in races.
![image](https://github.com/RPCS3/rpcs3/assets/137835937/cd79b93a-fbf1-497c-ad8b-3c9feba2e0d7)
Was able to finish a few races with this setting.
![image](https://github.com/RPCS3/rpcs3/assets/137835937/c7e39f86-01ae-4f75-915c-c6f7fcc1ba0f)
Log: [RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/12034176/RPCS3.log.gz)

**Comment by Ordinary205** [comment]
Using PPU Thread Count 1 didn't solve the issue for me.
![Need for Speed Prostreet](https://github.com/RPCS3/rpcs3/assets/125112932/7cbccd92-ce03-43c3-b8e2-9e7d0995f636)
[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/12062201/RPCS3.log.gz)

**Comment by cas-1092** [comment]
After further testing, I have found out that crashing happens only during certain events even with PPU Thread count set to 1.

Download my save file, [BLUS30066-SAVEDATA.zip](https://github.com/RPCS3/rpcs3/files/12062682/BLUS30066-SAVEDATA.zip) and open career mode.
Selecting this race, the grip class event, or any of the time attacks should crash your game.
![image](https://github.com/RPCS3/rpcs3/assets/137835937/696123f5-43c3-4159-aaad-26bf0db73c3f)
[GRIP EVENT RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/12062756/RPCS3.log.gz)

However, the drag event works just fine.
![image](https://github.com/RPCS3/rpcs3/assets/137835937/13f0fd39-cb6f-4276-b774-489e0f8d7db4)
[DRAG EVENT RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/12062741/RPCS3.log.gz)

**Comment by Ordinary205** [comment]
After doing some research, I found out that enabling PPU interpreter (static) fixes the crash, even when loading and during races, but reduces the performance.
![Need for Speed Prostreet](https://github.com/RPCS3/rpcs3/assets/125112932/f62469b6-0172-4928-8e50-002103953743)

Needs information if this works on other users or not.
[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/12184198/RPCS3.log.gz)

**Comment by cas-1092** [comment]
Works for me, using the savefile I mentioned earlier.
![image](https://github.com/RPCS3/rpcs3/assets/137835937/e328d63d-02cc-486f-9a69-27755fabeb57)
![image](https://github.com/RPCS3/rpcs3/assets/137835937/f931fcd9-3daf-443d-92bd-aec748ffee25)
[Need for Speed ProStreet [BLUS30066].log.gz](https://github.com/RPCS3/rpcs3/files/12184494/Need.for.Speed.ProStreet.BLUS30066.log.gz)

**Comment by Ordinary205** [comment]
I have figured out a better solution on how to fix this "Unknown STOP Crashes" without using PPU interpreter, is by enabling Atomic+Accurate SPU DMA+Accurate Cache Line Stores.
![NFS Prostreet 1](https://github.com/RPCS3/rpcs3/assets/125112932/fc160e7d-0eea-4896-a9ad-11a47f3059bb)
![NFS Prostreet 2](https://github.com/RPCS3/rpcs3/assets/125112932/4cb2516c-7389-4435-bd56-1f6ab975c0da)
![NFS Prostreet 3](https://github.com/RPCS3/rpcs3/assets/125112932/8bbd0eac-3cd7-45cc-b5cb-3b9fd4582698)
![NFS Prostreet 4](https://github.com/RPCS3/rpcs3/assets/125112932/ddb15dcf-b260-4aa8-99bb-622f92aeed84)
[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/13689194/RPCS3.log.gz)
Enabling these options may have a performance impact, but its much faster than PPU interpreter.

**Comment by cas-1092** [comment]
With these settings I was unable to enter the career, as it would either give me an spu access violation,
![image](https://github.com/RPCS3/rpcs3/assets/137835937/20f8e6ff-5368-4c7d-b41d-eb801b91e676)
or it would just freeze on the loading screen.
![image](https://github.com/RPCS3/rpcs3/assets/137835937/c6cbfa62-bae6-49f0-8350-63e9a4a81ac9)
Log: [RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/13689595/RPCS3.log.gz)

**Comment by superjeepboy** [comment]
updating title to reflect current situation, as it looks like it isn't just limited to just track crashes/save loads

**Comment by epee92370** [comment]
For who being interested , my screen plant , bug and crashing  eror said Its cannot oppening map and cars file and atempting   was cause my Power supply of my computer was not enough strong , my alimentation  gone down not long ago i buy  a knew one and no problem any more , take more watts like 100 thats its need .

**Comment by matheuswillder** [comment]
>my alimentation gone down not long ago i buy a knew one and no problem any more , take more watts like 100 thats its need .

These issues are not caused by a faulty PSU, but due to the current state of the emulation.

**Comment by esilvaolive** [comment]
I SOLVED THE PROBLEM
In this video, I demonstrate the only way to fix the crash with NFS Pro Street on RPCS3. With the settings I show at the beginning of the video, the game runs flawlessly in any race. The only downside is that the FPS count drops compared to the default settings.
[https://www.youtube.com/watch?v=Han-Jq3ef7o](url)

**Comment by ghost** [comment]
Just checking to see if this was forgotten about or solved

**Comment by kd-11** [comment]
100% reproducible when running under linux/steamos. The game cannot launch at all beyond menus.
